config/solium.json

```diff
@@ -120,6 +120,13 @@
 			"description": "Ensure that all strings use only 1 style - either double quotes or single quotes."
 		},
 
+		"value-in-payable": {
+			"enabled": true,
+			"recommended": true,
+			"type": "error",
+			"description": "Ensure that all strings use only 1 style - either double quotes or single quotes."
```

Comment:
I think you forgot to change the rule's description here (happens with me a lot!)